roles/postgres/vars/main.yml

```
@@ -0,0 +1,11 @@
+postgresql_version: "14"
+postgresql_daemon: "postgresql@{{ postgresql_version }}-main"
+postgres_user: "postgres"
+
+postgresql_users:
+  - name: joe
+    login_user: joe
+    role_attr_flags: SUPERUSER
+
+postgresql_databases:
+  - name: pythondiscord
```

Do we want to keep this? What would be stored on the database?

```suggestion
postgresql_databases: []
```